### ISSUE TITLE
make:migration fails and not giving good clues (Symfony 6.2)

### ISSUE THREAD

**Issue by fabpico** [symfony/maker-bundle#1280]
php:8.1.11

composer.json

````
{
  "require": {
    "symfony/runtime": "^6.2",
    "symfony/http-kernel": "^6.2",
    "symfony/framework-bundle": "^6.2",
    "symfony/console": "^6.2",
    "doctrine/doctrine-bundle": "^2.8",
    "symfony/orm-pack": "^2.3",
    "symfony/security-bundle": "^6.2"
  },
  "require-dev": {
    "phpunit/phpunit": "^9.5",
    "symfony/maker-bundle": "^1.48"
  },
  "autoload": {
    "psr-4": {
      "App\\": "src/"
    }
  },
  "config": {
    "allow-plugins": {
      "symfony/runtime": true
    }
  }
}
````
As described in https://symfony.com/doc/current/doctrine.html#migrations-creating-the-database-tables-schema, I do `php bin/console make:migration`. I receive:

````
[critical] Error thrown while running command "make:migration". Message: "There are no commands defined in the "doctrine:migrations" namespace.

Did you mean one of these?
    doctrine
    doctrine:cache
    doctrine:database
    doctrine:mapping
    doctrine:query
    doctrine:schema"
````
No, I did not mean some of those commands. I ran `php bin/console list make` to see if “make:migration” exists, and it exists.

Then I googled, and I found https://stackoverflow.com/questions/17004082/doctrine-migrations-there-are-no-commands-defined-in-the-namespace, and I thought maybe I should install DoctrineMigrationsBundle. I installed it, retried make:migration, and the command was found. So this is the first optimization needed, inform to install DoctrineMigrationsBundle.

Then when I run make:migration again, I receive:

````
[critical] Error thrown while running command "make:migration". Message: "assert(is_string($namespace))"

In DiffCommand.php line 136:
                                 
  assert(is_string($namespace))

````
With this message I have no clue what the problem may be, I am forced to debug deep in the maker bundle.

**Comment by fabpico** [comment]
Traced back that it has to do with configuring `migrations_paths` . With googling I found https://symfony.com/bundles/DoctrineMigrationsBundle/current/index.html#configuration, and it says

> If you use Symfony Flex, the doctrine_migrations.yaml config file is created automatically.

But I don't use Flex.

After configuring at least 

````
# config/packages/doctrine_migrations.yaml
doctrine_migrations:
  migrations_paths:
    'DoctrineMigrations': '%kernel.project_dir%/migrations'
````
The workflow proceeds to another (a more) descriptive error. 

I suggest that `make:migration` checks if doctrine_migrations.yaml is existing, and configured properly, and maybe also suggests a compatible example configuration.

**Comment by jrushlow** [comment]
Howdy! `make:migration` is essentially a wrapper around `doctrine:migrations:diff` - we've recently merged #1276 to be a bit more helpful when `DoctrineMigrationsBundle` is missing. But, we do not have any plans at the moment to add additional configuration checks doctrine related settings. However, PR's are always welcome.

Cheers!
